egs/voxpopuli/ASR/prepare.sh

```diff
@@ -0,0 +1,263 @@
+#!/usr/bin/env bash
+
+. /mnt/matylda5/iveselyk/ASR_TOOLKITS/K2_SHERPA_PYTORCH20/conda-activate.sh
```

Comment:
Please add the following two lines
https://github.com/k2-fsa/icefall/blob/6d275ddf9fdd67a32b79b93d70fedffe4b156d5c/egs/librispeech/ASR/prepare.sh#L3-L4